plans/dht/README.md

```diff
@@ -72,3 +79,23 @@ IPFS can safely rely on the latest DHT upgrades by running go-libp2p DHT tests d
     - Nodes ran 5 random-walk queries to populate their Routing Tables
   - **Act I**
     - Each node calls Provide for `i-provides` until it reaches a total of `n-provides`
+
+### `Test:` Provider record saturation

```

Comment:
This plan will benefit from having a controlled network (i.e. position the nodes in known locations in the network). For that, I'll have to make sure I've PeerIds that are evenly distributed over the hash address.

Reply:
Note to self: use https://github.com/libp2p/go-libp2p-kbucket/blob/master/bucket_prefixmap.go for that